package.json

```diff
@@ -10,6 +10,8 @@
   "license": "MIT",
   "main": "source/index.js",
   "scripts": {
+    "build": "./node_modules/.bin/gulp build",
```

Comment:
This is also related to the `codestyle` line which I didn’t notice in the PR so feel free to add additional commit that updates the line if you’d like to.